src/styles/blocks/about.scss

```diff
@@ -0,0 +1,45 @@
+.about {
+  &__title {
+    margin: 0 0 32px;
+
+    font-weight: $font-weight-bold;
+    font-size: 32px;
+    line-height: 32px;
+    text-align: center;
+    letter-spacing: -1.5px;
+    color: $primary-font-color;
+
+    @include onTablet {
+      margin-bottom: 0;
+
+      font-size: 40px;
+      line-height: 40px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 0;
+
```

Comment:
```suggestion
```

Duplicate prop. It's already set for tablets. The desktop view will already have it.